src/test/java/com/github/tomakehurst/wiremock/AdminApiTest.java

```diff
@@ -1372,6 +1372,72 @@ void getVersionRequestReturnsTextBodyWhenAcceptHeaderIsTextPlain() {
     assertThat(response.content(), is("X.X.X"));
   }
 
+  @Test
+  void findUnmatchedStubMappingsWithNoMappings() {
```

Comment:
IMO it's probably best to split this up into separate test files for each endpoint or group (e.g. `AdminApiMappingsTest`, `AdminApiRequestsTest`, etc), since it's getting a bit hard to follow. I'm not sure where to put them, though. Maybe `src/test/java/com/github/tomakehurst/wiremock/admin/acceptance`?

Reply:
Perhaps we could do that as a separate refactor PR?

Reply:
I could, but I don't know where to put these tests.